lib/pusher.js

```diff
@@ -10,6 +10,21 @@ var Config = require('./config');
 var Token = require('./token');
 var WebHook = require('./webhook');
 
+var validateChannel = function(channel) {
+  if (typeof channel !== "string" || channel === "" || channel.match(/[^A-Za-z0-9_\-=@,.;]/)) {
+    throw new Error("Invalid channel name: '" + channel + "'");
+  }
+  if (channel.length > 200) {
+    throw new Error("Channel name too long: '" + channel + "'");
+  }
+}
+
+var validateSocketId = function(socketId) {
+  if (typeof socketId !== "string" || socketId === "" || !socketId.match(/^\d+\.\d+$/)) {
```

Comment:
Is @zimbatm's comment about the required regexp [here](https://github.com/pusher/pusher-http-python/commit/afaf75d271ad447df215e9b32448f11b9565792a#commitcomment-11131227) relevant?